Tests/Tests/TooltipDispatcherSpec.swift

```diff
@@ -204,6 +207,13 @@ class TooltipDispatcherSpec: QuickSpec {
                         router.completeDisplayingTooltip(cancelled: false)
                         expect(campaignRepository.incrementImpressionsCalls).toAfterTimeout(equal(0))
                     }
+
+                    it("will perform ping") {
```

Comment:
I don't think this test case is related to 'contexts' feature. Please create a new context in `when delegate is set` context level

Reply:
Added here [fd45717](https://github.com/rakutentech/ios-inappmessaging/pull/252/commits/4a6f6859f527d9da77adbc56c71a2e458fd45717).